crates/macro/ui-tests/invalid-imports.stderr

```diff
@@ -1,9 +1,3 @@
-error: it is currently not sound to use lifetimes in function signatures
- --> ui-tests/invalid-imports.rs:7:16
-  |
-7 |     fn f() -> &'static u32;
-  |                ^^^^^^^
```

Comment:
It's a little annoying that this seems to be getting suppressed by the other errors, so the new error ('cannot return references in #[wasm_bindgen] imports yet') isn't being tested.

Reply:
Separated the tests.